packages/components/src/sandbox/index.js

```diff
@@ -48,7 +48,7 @@ class Sandbox extends Component {
 		if ( 'string' === typeof data ) {
 			try {
 				data = JSON.parse( data );
-			} catch ( e ) {} // eslint-disable-line no-empty
```

Comment:
Did we disable this rule? I'm mostly curious.

Reply:
It probably has never been enabled in the first place. I don't see any references to that rule in the ESLint configs, nor do I see one to `eslint:recommended`, which includes that rule.